smsdk/client.py

```diff
@@ -312,14 +338,17 @@ def get_kpis_for_asset(self, **kwargs):
 
         return kpis(self.session, base_url).get_kpis_for_asset(**kwargs)
 
-    def get_kpi_data_viz(
-        self,
-        machine_sources=None,
-        kpis=None,
-        i_vars=None,
-        time_selection=None,
-        **kwargs,
-    ):
+    def get_kpi_data_viz(self, *args, **kwargs):
```

Comment:
Why did you get rid of the named arguments.  Seems like you could left it in especially since they were all already set to =None.   Honestly looking at this function I don't think this change actually did anything but remove some functionality and we really just needed the documentation update.

Reply:
The `*args` and `**kwargs` parameters capture all positional and keyword arguments, respectively. I have not modified the existing functionality of the code; I have merely altered the style of argument passing.